pyproject.toml

```diff
@@ -6,7 +6,7 @@ version = "1.0.0"
 authors = [
     { name = "Christian Clauss", email = "cclauss@me.com" },
     { name = "Dhruv Manilawala", email = "dhruvmanila@gmail.com" },
-    { name = "John law", email = "dhruvmanila@gmail.com" }
+    { name = "John Law", email = "dhruvmanila@gmail.com" }
```

Comment:
The email address is wrong.

Reply:
> The email address is wrong.

What is the correct email address?
I took it from his github bio

Reply:
@poyea Do you want to be listed here?  With your email address?

Reply:
I'm good with that